accepted/2021/net6.0-tfms/net6.0-tfms.md

```diff
@@ -0,0 +1,110 @@
+# .NET 6.0 Target Frameworks
+
+**PM** [Immo Landwerth](https://github.com/terrajobst)
+
+In [.NET 5.0][net5.0] we defined a new syntax for target frameworks (TFM) that
+allows us to avoids tools and humans to use a decoder ring, such as the .NET
+Standard version table, to figure out what is compatible with what. In this
+document, we're listing which TFMs we're adding in .NET 6.0.
+
+## Scenarios and User Experience
+
+TBD
+
+## Requirements
+
+### Goals
+
+* Support the definition of frameworks relevant for Xamarin development
+
+### Non-Goals
+
+* Re-design the TFM syntax
+
+## Stakeholders and Reviewers
+
+* MSBuild / SDK team
+* NuGet team
+* Project system team
+* Xamarin team
+
+## Design
+
+| TFM                | Compatible With                          |
+|--------------------|------------------------------------------|
+| net6.0             | (subsequent version of net5.0)           |
+| net6.0-windows     | (subsequent version of net5.0-windows)   |
+| net6.0-android     | xamarin.android                          |
+|                    | (+everything else inherited from net6.0) |
+| net6.0-ios         | xamarin.ios                              |
+|                    | (+everything else inherited from net6.0) |
+| net6.0-macos       | xamarin.mac                              |
+|                    | (+everything else inherited from net6.0) |
+| net6.0-maccatalyst | xamarin.ios                              |
+|                    | (+everything else inherited from net6.0) |
+| net6.0-tvos        | xamarin.tvos                             |
+|                    | (+everything else inherited from net6.0) |
+| net6.0-watchos     | xamarin.watchos                          |
```

Comment:
if we do not plan on supporting watchOS then we should not document a TFM for it
c.c. @davidortinau

Reply:
Makes sense, will remove watchOS.